### PR TITLE
add boulder story

### PR DESCRIPTION
**Pull request by jimprince**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/find-exit/into-mountains/mountains.md [hunk 1]
```diff
@@ -10,4 +10,6 @@ Do you:
 
 [Continue upwards?](./upwards/upwards.md)
 
+[Push boulders into the ravine.](./boulder/boulder.md)
```
Comment (casey-collab) [inline english/find-exit/into-mountains/mountains.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/find-exit/into-mountains/boulder/boulders.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+As you push against some of the boulders, they fall downwards into the
```
Comment (casey-collab) [inline english/find-exit/into-mountains/boulder/boulders.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jimprince** [comment]
Thanks! I think I got them now.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :penguin:

**Pull request merged**